tests/testthat/test-operators.R

```diff
@@ -0,0 +1,142 @@
+test_that("param operators", {
+  data(example_scan)
+  example_param <- get_param(example_scan, "VRADH")
+  expect_visible(example_param + 1)
+  expect_equal(example_param + 1, 1 + example_param)
+  expect_equal(example_param + example_param, 2 * example_param)
+  expect_equal(example_param + -1, example_param - 1)
+  expect_equal(example_param / 2, example_param * .5)
+  expect_equal(example_param * 1:480, 1:480 * example_param)
+  expect_equal(matrix(example_param) * 1:480, matrix(1:480 * example_param))
+})
+
+test_that("param and scan errors", {
+  p1 <- structure(1:12, .Dim = c(3L, 4L), class = c("param", "matrix", "array"), radar = "SE50", datetime = structure(1445191200, class = c("POSIXct", "POSIXt"), tzone = "UTC"), geo = list(lat = 56.3675003051758, lon = 12.8516998291016, height = 209, elangle = 0.5, rscale = 500, ascale = 1, rstart = 0), param = "VRADH")
```

Comment:
Please base all the test on (modifications to) the example_scan and example_pvol objects that come with the package, instead of building up new objects from scratch. Objects might change in the future (e.g. new attributes), and we want to make sure these operations still behave properly after those updates.

Reply:
Good point, the latest commit should solve this